### PR TITLE
Development

### PR DESCRIPTION
**Pull request by trox115**

### CODE REVIEW

**Comment by HenryTabima** [comment]
# Great job :+1: 
Hey! I like your work on the navbar :clap:. Good use of semantic HTML, and nice touch with the favicon, you're on the right path :runner:. Just some changes before you get thumbs up on this project:

## General

- [x] Your Readme is already good looking. let's rewrite it using markdown. it's easy to learn, write and read. https://guides.github.com/features/mastering-markdown/

## Background
- [x]  The main content should be wrapped around `<main>` tags and have an image set as background

## Navbar
- [x]  It should be within the `<main>` tag

## Search field
- [x] The search field should be an `<input>` HTML tag with the *focus* effect enabled

## Main content links

- [x] There should be three links
- [x] Each link is an `<a>` HTML tag, with a `<div>` tag inside it.
- [x] The `<div>` tag  inside the `<a>` tag should use an image as background -- [ ] The text on the original page is actually an image

## Featured links (before the footer)
- [x]  The top part is made of four `<a>` tags with `<div>` tags inside it
- [x]  The `<div>`s inside the`<a>` tags should use an image as a background
- [x] Using *display flex* for positioning links is recommended. Make the 4 elements fit the available space

### Happy coding :computer:

File: style.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  font-family: Roboto;
+}
+.text a {
+  color: white;
+}
+.text a:hover {
+  color: white;
+}
+
+.im {
```
Comment (HenryTabima) [inline style.css]: use complete and meaningful class names
> why: in larger projects short or non-meaningful class names are hare to debug

**Comment by HenryTabima** [comment]
Excellent! :1st_place_medal: 
You did a great job on the requirements I asked you for. One last change before giving you the thumbs up.
- the featured links don't fit the width in different screens (see the next comments for recommendations)

File: style.css [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+}
+
+
+.images-bot {
+  display: flex;
+  width: 100%;
+  height: 200px;
+  flex-wrap:nowrap;
+}
+
```
Comment (HenryTabima) [inline style.css:132]: you can add a selector to the `a` tags inside the `.images-bot` element, and add the property `flex: 1`. Then add another selector to the `div`s inside the previous `a` tags and set the next properties `background-size: cover`, `width: 100%` and `height: 100%`. Finally, remove the fixed widths and heights in the `#promo` ids

**Comment by Torres-ssf** [comment]
### Great job @trox115 
I will give this project a 👍 
Go on, merge it and don't forget to submit the form.

**Pull request merged**